apps/acrs/fields/index.js

```diff
@@ -20,6 +20,31 @@ module.exports = {
       className: 'visuallyhidden'
     }
   },
+  'referrer-email-options': {
+    mixin: 'radio-group',
+    options: [
+      {
+        value: 'yes',
+        toggle: 'referrer-email-address',
+        child: 'input-text'
+      },
+      {
+        value: 'no'
+      }
+    ],
+    validate: 'required',
+    legend: {
+      className: 'visuallyhidden'
+    }
+  },
+  'referrer-email-address': {
+    dependent: {
+      field: 'referrer-email-options',
+      value: 'yes'
+    },
+    validate: ['required', 'email', { type: 'maxlength', arguments: 254 }]
+    // validate: ['email', { type: 'maxlength', arguments: 254 }, { type: 'minlength', arguments: 6 }]
```

Comment:
If you want to keep this no worries, if you doubt you'll use this in the end I'd remove the comment.

Reply:
I'll remove. I couldn't figure out if `minlength` is supported when type is `email`